### DISCUSSION TITLE
Support Custom roms without Google services

### DISCUSSION THREAD

**Discussion by alr86** [community/community#111425] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

Github mobile login in Custom Android roms without Google services get crash. Please support de-googled phones or microG at least.

**Comment by stevepopovich** [comment]
Hi @alr86 Unfortunately, we are not able to support this use case, due to security concerns.  We advise you download the app from the Play Store only. Thanks for your understanding!

**Comment by Pham1993** [comment]
Hi

**Comment by alr86** [comment, marked as answer]
✅ App work fine on de-googled devices with micro-g, just need a chromium base browser like chrome or vivaldi(Opera mobile doesn't work)
